qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -327,11 +203,29 @@ def get(self, ignore):
             study_proc = proc_samples = None
         info = _build_study_info(self.current_user, study_proc=study_proc,
                                  proc_samples=proc_samples)
+        # linkifying data
```

Comment:
I don't have strong feeling about this. Note that we have tests for this and _build_study_info so we are testing both the formatted and not formatted lists.